### PR TITLE
Allow overriding UI of checkout by payment handler

### PR DESCRIPTION
**Pull request by Kukks**
Taken off #920 / #971

### CODE REVIEW

File: BTCPayServer/Views/Invoice/CheckoutNoScript.cshtml [hunk 1]
```diff
@@ -16,16 +16,22 @@
...
-            <p><a href="@Model.InvoiceBitcoinUrl" style="word-break: break-word;">@Model.InvoiceBitcoinUrl</a></p>
-            @if (Model.IsLightning)
-            {
-                <p>Peer Info: <b>@Model.PeerInfo</b></p>
-            }
-        </div>
-
+        if (!string.IsNullOrEmpty(Model.UISettings?.NoScriptPartialName))
+        {
+            <partial model="@Model" name="@Model.UISettings.NoScriptPartialName"/>
```
Comment (NicolasDorier) [inline BTCPayServer/Views/Invoice/CheckoutNoScript.cshtml]: while you are at it, maybe you can move the else part in a partial?

Comment (Kukks) [reply BTCPayServer/Views/Invoice/CheckoutNoScript.cshtml]: Already on it in a separate PR( I am migrating code to vue from the legacy jquery stuff)

Comment (NicolasDorier) [reply BTCPayServer/Views/Invoice/CheckoutNoScript.cshtml]: should you do it in this PR so that's done?

Comment (Kukks) [reply BTCPayServer/Views/Invoice/CheckoutNoScript.cshtml]: Some of the vue migration has to happen for the move to the partial so I think separate PR is better

**Pull request merged**